lib/text/ttml_text_parser.js

```diff
@@ -312,8 +312,8 @@ shaka.text.TtmlTextParser = class {
           shaka.util.Error.Code.INVALID_TEXT_CUE);
     }
 
-    start += offset;
-    end += offset;
+    start = isNested && start == null ? 0 : start + offset;
+    end = isNested && end == null ? 0 : end + offset;
```

Comment:
Could you please add parentheses, so it's easier to read? Thank you!

Reply:
Thanks, I've pushed an update.